### ISSUE TITLE
Add support for symlinks in files

### ISSUE THREAD

**Issue by schomatis** [ipfs/go-mfs#4]
See https://github.com/ipfs/go-ipfs/issues/5302.

**Comment by aschmahmann** [comment]
This repository is no longer maintained and has been copied over to [Boxo](https://github.com/ipfs/boxo).  In an effort to avoid noise and crippling in the Boxo repo from the weight of issues of the past, we are closing most issues and PRs in this repo.  Please feel free to open a new issue in Boxo (and reference this issue) if resolving this issue is still critical for unblocking or improving your usecase.

You can learn more in the [FAQs for the Boxo repo copying/consolidation effort](https://github.com/ipfs/boxo/wiki/Copied-or-Migrated-Repos-FAQ).
